src/median.test.ts

```diff
@@ -0,0 +1,26 @@
+import { median } from "./median";
+import { pipe } from "./pipe";
+
+describe("dataFirst", () => {
+  it("should return the median of numbers in an array", () => {
+    expect(median([6, 10, 11])).toBe(10);
+    expect(median([1, 2, 3, 9])).toBe(2.5);
+    expect(median([-1, 0, 1, 10])).toBe(0.5);
+  });
```

Comment:
Because the logic is different for arrays of even length and arrays of odd length, let's split the tests so that each branch is tested separately. This would make the tests more precise in signaling what broke.
```suggestion
  test("arrays of odd length", () => {
    expect(median([6, 10, 11])).toBe(10);
  });
  
  test("arrays of even length", () => {
    expect(median([1, 2, 3, 9])).toBe(2.5);
    expect(median([-1, 0, 1, 10])).toBe(0.5);
  });
```